client/src/components/Chat.jsx

```diff
@@ -589,13 +599,15 @@ const Chat = () => {
                 className="flex justify-center items-center mt-[40px]"
                 onSubmit={handleSubmit}
             >
-                <div className="w-full flex items-center justify-between bg-secondary rounded-l-md">
-                    <textarea
-                        placeholder="Send a Message....."
-                        className="h-[45px] focus:outline-none w-[96%] bg-secondary text-white rounded-[15px] resize-none pl-[22px] pr-[22px] text-[18px] placeholder-shown:align-middle"
-                        ref={inputRef}
-                        onChange={handleTypingStatus}
-                    />
+                  <div className="w-full flex items-center justify-between bg-secondary rounded-l-md" id='message-input-container'>
+                <textarea
+                    placeholder="Send a Message..."
+                    id="message-input"
+                    className="h-[45px] focus:outline-none w-[96%] bg-secondary text-white rounded-[15px] resize-none pl-[22px] pr-[22px] py-[10px] text-[16px] placeholder-shown:align-middle"
+                    ref={inputRef}
+                    value={message}
```

Comment:
i don't think this concerns your assigned issue

Reply:
I kinda needed to calculate the message inputted and adjusted the placeholder text position using py